src/test/java/softvisionProject/POM/CareersPage.java

```diff
@@ -37,28 +37,48 @@ public CareersPage(WebDriver driver) {
     //
 
 
-    // HashMap methods
+    // HashMap methods - get the LOCATION / STUDIOS drop down list items
     public void selectLocations(String name) {
         if (locationz.isDisplayed()) {
             System.out.println("- location display: pass");
             locationz.click();
             System.out.println("location display clicked");
             JavaMap test = new JavaMap();
             HashMap <String, WebElement> dropDownList = test.getDropList(driver);
-            for (String location: dropDownList.keySet()) {
-                if (location.contains(name)) {
-                System.out.println("location by name found");
-                System.out.println("Name taken" + name);
-                dropDownList.get(location).click();
-                break;
-                } else {
-                System.out.println("Nothing to click on");
-            }
-            }
+                for (String location: dropDownList.keySet()) {
+                    if (location.contains(name)) {
+                    System.out.println("location by name found");
+                    System.out.println("Name taken " + name);
+                    dropDownList.get(location).click();
+                    break;
+                    } else {
+                    System.out.println("Nothing to click on");
+                    }
+                }
         } else {
                 System.out.println("- location display: fail");
         }
     }
+
+    // HashMap methods - get the EXPERTISE / GUILDS drop down list items
+    public void selectExpertise(String name) {
+        if (expertisez.isDisplayed()) {
+            System.out.println("- expertise display: pass");
+            expertisez.click();
+            System.out.println("- expertise display clicked");
+            JavaMap test = new JavaMap();
+            HashMap <String, WebElement> expertiseList = test.getExpertiseList(driver);
+                for(String expertise: expertiseList.keySet()) {
+                    if (expertise.contains(name)) {
+                        System.out.println("Expertise found : " + name);
+                        expertiseList.get(expertise).click();
+                        break;
+                    } else {
+                    System.out.println("Nothing to click on");
```

Comment:
expecting a specific message for the user. "Nothing to click on" do not tell me too much

Reply:
Changed. Will update shortly.